SalesforceCommerceCloud/pwa-kit
# Implement shopper products

# Description
This PR implemented `useShopperCategories`, `useShopperCategory`,`useShopperProduct`, `useShopperProduct`



# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes


# How to Test-Drive This PR

-  Pull the code
- npm ci
- cd to the test project, npm start
- Before clicking on any links, please grab a token in your console and put it inside your `Provider` [here](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/681/files#diff-b4f10b518ebe536f7c86763137c359e445247960b9c54a7597e78590f20faa91R58)
- Reload and see no error in the console, which means you are logged in as guest
- Navigate around to see the hooks in actions!

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
